include/blackhole/detail/sink/asynchronous.hpp

```diff
@@ -0,0 +1,80 @@
+#include <atomic>
+#include <thread>
+
+#include <cds/container/vyukov_mpmc_cycle_queue.h>
+
+#include "blackhole/sink.hpp"
+
+#include "blackhole/detail/recordbuf.hpp"
+
+namespace blackhole {
+inline namespace v1 {
+namespace experimental {
+namespace sink {
+
+using detail::recordbuf_t;
+
+/// I can imagine: drop, sleep, wait.
+class overflow_policy_t {
+public:
+    enum class action_t {
```

Comment:
How does this used?
I see only interface and factory for policy accepting string